samples/boards/reel_board/mesh_badge/src/mesh.c

```diff
@@ -154,7 +154,7 @@ static void gen_onoff_set_unack(struct bt_mesh_model *model,
 	       bt_mesh_model_elem(model)->addr, state->current);
 
 	/* Pin set low turns on LED's on the reel board */
-	if (set_led_state(state->dev_id, onoff)) {
+	if (set_led_state(state->dev_id, !onoff)) {
```

Comment:
The mapping from on/off to the matching GPIO value should be internal to the set_led_state function (otherwise the function would be called gpio-something).